lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -651,6 +827,7 @@ def __init__(self, config):
         self.scratch = config['scratch']
         self.callback_url = os.environ['SDK_CALLBACK_URL']
         self.ws_url = config['workspace-url']
+        self.token_user = os.environ['KB_AUTH_TOKEN'].split('client_id=')[1].split('|')[0]
```

Comment:
You can set up ctx any way you want in the tests: https://github.com/kbaseapps/ReadsUtils/blob/master/test/ReadsUtils_server_test.py#L47

The user id is set when running normally: https://github.com/kbaseapps/ReadsUtils/blob/master/lib/ReadsUtils/ReadsUtilsServer.py#L567